xwiki-platform-core/xwiki-platform-menu/xwiki-platform-menu-ui/src/main/resources/Menu/MenuTranslations.xml

```diff
@@ -84,6 +84,10 @@ rendering.macro.menu.parameter.type.name=Type
 rendering.macro.menu.parameter.type.description=The optional menu type determines the menu appearance and behaviour. The supported values are: horizontal (default value) and vertical.
 rendering.macro.menu.content.description=Define the menu structure using wiki syntax. Each menu item should be a list item and should contain the menu item label or link. You can use nested lists for sub-menu items.
 
+# Accessibility labels
+rendering.macro.menu.horizontal.label=Horizontal custom menu
+rendering.macro.menu.horizontal.label=Vertical custom menu
```

Comment:
Wouldn't it be better to just use this as a fallback and let the user provide a label to the macro? Also, when the menu macro is used twice, the labels should definitely be different.

Reply:
I'm worried that the user might not give a name descriptive enough to the menus, so I can't just use it as is.
I could however add a parameter in those entries to contain the name of the menu itself, this would make the labels different from one menu to the other (except for vertical menus where the admin can use the panel multiple times :thinking: ) and give a bit of extra info without relying too much on admin's practices when it comes to naming custom menus.
e.g.:
`rendering.macro.menu.horizontal.label=Horizontal custom menu: {0}`

Reply:
For example, on XWiki.org I think this menu macro is used for the main menu at the top. I think it would be strange to have the label "Horizontal custom menu" on this main navigation menu, regardless if there is also another label behind it.

Reply:
@michitux What label do you think would be more appropriate for this kind of navigation element?
Is it the "custom" which is inappropriate because this is not customizable for the user but only for admins?
Maybe "horizontal" is too close to the implementation and should be replaced by "top" (what the user sees).
I think keeping "menu" is relevant since this element is part of the menu macro.

Some possible alternatives: 
* "Top menu bar"
* "Top navigation menu"
* "Top navigation bar"


For reference:
https://www.xwiki.org/xwiki/bin/view/Documentation/UserGuide/Features/SecondGenerationWiki/
![19450ExampleMenu2](https://user-images.githubusercontent.com/28761965/222143875-bf30e8b2-ef97-422a-bf3d-12bc13bd66b8.png)
Iceberg with top menu:
![19450ExampleMenu1](https://user-images.githubusercontent.com/28761965/222143881-2cb55b14-1162-46f1-a606-0ed3398bb99d.png)

Reply:
> I'm worried that the user might not give a name descriptive enough to the menus, so I can't just use it as is.

I don't agree with this. The description of the label parameter should provide enough guidance to pick a suitable label. Also, I'm not sure how much value a prefix "Vertical navigation menu" provides to the user of a screen reader. I have the impression this might just be annoying, in particular if what comes afterwards doesn't expect the prefix as the admin won't be aware of the prefix (the description of the parameter doesn't even mention it). So you might end up with "Top navigation menu: Top navigation".